LabeledIconButton/LabeledIconButton.module.less

```diff
@@ -15,8 +15,31 @@
 		/* Public Class Names */
 	}
 
+	.label {
+		text-overflow: ellipsis;
```

Comment:
Could you find a way to show 2 line and apply ellipsis??
It seems impossible, but I'm not sure.

Reply:
I used the "line-clamp" property, and it seems to work only if I add a third word/character to the label. Meaning, I added a " !" at the end of the label in the storybook so that the ellipsis is applied. If I remove the " !" or just stick it to "Hello LabeledIconButton!" so that it doesn't count it as a different word, the label is just cut off, with no ellipsis dots. 
Do you think we should leave it like this or revert to the solution above?
Thanks!

Reply:
I think the previous one was better in terms of completeness. And as I talked to the GUI person, it seems to be changed to one line in v2.5. Let's go back to one line.